daemon/cluster/executor/container/container.go

```diff
@@ -207,7 +206,7 @@ func getMountMask(m *api.Mount) string {
 func (c *containerConfig) hostConfig() *enginecontainer.HostConfig {
 	return &enginecontainer.HostConfig{
 		Resources: c.resources(),
-		Binds:     c.bindMounts(),
+		Binds:     append(c.bindMounts(), c.volumes()...),
```

Comment:
It fills in the role of an anonymous volume. It's a use case we support today and I'm not sure we can actually leave out.